Assets/Minigames/Fight/Scripts/Behavior/Movement/BasicAIMovement.cs

```diff
@@ -47,40 +60,62 @@ private void Start()
     {
         // idiot proofing
         rb.gravityScale = 0;
-        // by default calculate a new path ever .5 seconds to adapt to target moving (causes slightly delayed pathing for fast moving objects but checking every frame causes jittering)
-        InvokeRepeating(updatePath, 0, 0.5f);
+        StartCoroutine(RepeatUpdatePath());
+    }
+    private IEnumerator RepeatUpdatePath()
+    {
+        while (true)
```

Comment:
i want to re-work this eventually. I have found a better way at work, we can review this together